Vagrantfile

```diff
@@ -367,7 +367,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             end
 
             node.vm.provision "shell" do |s|
-                s.inline = "echo '#{node_ips[0]} netmaster' >> /etc/hosts; echo '#{node_ips[1]} netmaster' >> /etc/hosts;   echo '#{node_addr} #{node_name}' >> /etc/hosts"
+               if ENV["DOCKER_SWARM"] && ENV["DOCKER_SWARM"] == "SWARM_MODE"
```

Comment:
You can just do:

```rb
ENV["DOCKER_SWARM"] == "SWARM_MODE"
```

`ENV` is a `Hash`, and `Hash#[]` returns nil when a key is not found.  If you want a `KeyError` when a key is missing, use `Hash#fetch`